pkg/conversion/cloner.go

```diff
@@ -153,6 +153,36 @@ func (c *Cloner) deepCopy(src reflect.Value) (reflect.Value, error) {
 		err := fv(src.Interface(), outValue.Interface(), c)
 		return outValue, err
 	}
+
+	// Check for custom deep copy funcs on the pointer of this type
+	if src.Kind() == reflect.Struct {
```

Comment:
This case is slow and if the code path is used, it's a (performance) bug in the code. We should at least `glog.Errorf` here. Background: go copies the struct on every function call. This sums up and costs.